pegasus/forms/hoc_signup_2020.rb

```diff
@@ -2,6 +2,39 @@
 
 #HOC Sign up form for 2020. 2017 was the last form with new logic
 class HocSignup2020 < HocSignup2017
+  def self.normalize(data)
+    Honeybadger.context({data: data})
+    result = {}
+    result[:email_s] = required email_address data[:email_s]
+    result[:name_s] = required stripped data[:name_s]
+    result[:organization_name_s] = stripped data[:organization_name_s]
+    result[:event_type_s] = required enum(data[:event_type_s].to_s.strip.downcase, ['in_school', 'out_of_school', 'at_home'])
```

Comment:
this function used to be inherited from the previous HocSignup years. I had to copy it over and update this line so we don't fail validation due to the new 'at_home' event type.